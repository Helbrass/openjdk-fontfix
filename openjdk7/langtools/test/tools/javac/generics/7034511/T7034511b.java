/*
 * @test /nodynamiccopyright/
 * @ignore backing out 7034511, see 7040883
 * @bug     7034511 7040883
 * @summary Loophole in typesafety
 * @compile/fail/ref=T7034511b.out -XDrawDiagnostics T7034511b.java
 */

class T7034511b {
    static class MyList<E> {
        E toArray(E[] e) { return null; }
    }

    void test(MyList<?> ml, Object o[]) {
        ml.toArray(o);
    }
}
