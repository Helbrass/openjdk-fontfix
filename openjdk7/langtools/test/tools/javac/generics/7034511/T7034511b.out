T7034511b.java:14:11: compiler.err.cant.apply.symbol.1: kindname.method, toArray, compiler.misc.type.captureof: 1, ?[], java.lang.Object[], kindname.class, T7034511b.MyList<E>, (compiler.misc.no.conforming.assignment.exists: java.lang.Object[], compiler.misc.type.captureof: 1, ?[])
1 error
